hie.yaml

```diff
@@ -1 +1,3 @@
 cradle: {stack: {component: "ghcide:lib"}}
+dependencies:
+  - ghcide.cabal
```

Comment:
Hmm, I think hie-bios should auto-detect this as a dependency.

Reply:
hie-bios gives package.yaml and stack.yaml, but not .cabal files. I think it makes sense because in most stack-generated projects, the .cabal file is generated from package.yaml.

Reply:
https://github.com/mpickering/hie-bios/blob/master/src/HIE/Bios/Cradle.hs#L486
Nope, I think it is a bug. You are the first to actually use that feature, thus it makes sense that you are discovering bugs :(

Reply:
I'm not actually sure I checked this. Let me check now.

Reply:
I was wrong. ghcide.cabal file is also picked up automatically by hie-bios. Reverted. Thanks, @fendor !